### ISSUE TITLE
HTML report documentation

### ISSUE THREAD

**Issue by AntoniosBarotsis** [bheisler/criterion.rs#638]
I was trying to benchmark some of my code and could not get the HTML report generated. I wasn't sure why up until I checked a previous project I worked on which specified the `html_reports` feature.

I feel like mentioning this in [here](https://github.com/bheisler/criterion.rs/blob/master/book/src/user_guide/html_report.md) would probably help any newcomers to the project. I could make a small PR for this, just let me know what you think of this!

I would add something like 

"Criterion.rs can generate an HTML report displaying the results of the benchmark under
`target/criterion/reports/index.html`. **Note that for this to happen you need to include the `html_reports` like so `criterion = { version = "0.4.0", features = [ "html_reports" ]}`**.`"

**Comment by conradoplg** [comment]
I just got bit by this but figured that the solution was to explicitly pass `--plotting-backend gnuplot` (which annoyingly [requires you to specify the benchmark to run](https://bheisler.github.io/criterion.rs/book/faq.html#cargo-bench-gives-unrecognized-option-errors-for-valid-command-line-options), otherwise it tries to pass that flag to the library benchark which won't use criterion and it will fail; e.g. `cargo bench --bench bench -- --plotting-backend gnuplot`)

Investigating the source code I found out that `html_reports` just enables the HTML report generation by default, and not the _ability_ to generate HTML reports which was what I assumed. So this is why both solutions work.

In short, yeah, the documentation could be clearer.

**Comment by AntoniosBarotsis** [comment]
@bheisler I'll just leave this up to you :)

**Comment by BartTerpstra** [comment]
got stuck on this as well.
this was the answer. 
thank you :+1:

**Comment by bazhenov** [comment]
Same situation. It would be nice to have this sort of information in the manual.
